apps/SageAccountWeb/index.html

```diff
@@ -31,16 +31,7 @@
         })(window,document,'script','dataLayer','GTM-KPW4KS62');
       </script>
       <!-- End Google Tag Manager -->
-    
-    <script type="module">
```

Comment:
Verified that SynapseForm still works in production/build mode after removing this from portals.

This polyfill has likely not been necessary for a while, as `vite-plugin-node-polyfills` has handled pretty much all cases`.